packaging/opae/deb/control

```diff
@@ -39,6 +39,6 @@ Priority: optional
 Architecture: any
 Multi-Arch: same
 Depends: opae (= ${binary:Version}), opae-devel (= ${binary:Version}),
-         libhwloc, libtbb, libedit,
+         libhwloc-dev, libtbb-dev, libedit-dev,
```

Comment:
I'm on the fence about this change.  To my understanding, Build-Depends (above) is to contain the devel packages needed to create the .deb, but Depends is to contain the runtime dependencies, which would be the non-devel packages.